doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
 . Unzip the downloaded archive.
 . Move the `{PRODUCT_FILE}-process-migration-service-standalone.jar` file from the add-ons archive to a desired location.
-. In the location, create a YAML file containing the kieserver and Thorntail configuration, for example:
+. In the location, create a YAML file containing the {KIE_VERSION} and Quarkus configuration, for example:
 +
 ----
-thorntail:
-  deployment:
-    process-migration.war:
-      jaxrs:
-        application-path: /rest
-      web:
-        login-config:
-          auth-method: BASIC
-          security-domain: pim
-        security-constraints:
-          - url-pattern: /*
-            roles: [ admin ]
-          - url-pattern: /health/*
-  datasources:
-    data-sources:
-      pimDS:
-        driver-name: h2
-        connection-url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-        user-name: DS_USERNAME
-        password: DS_PASSWORD
+quarkus:
+  class-loading:
+    removed-artifacts: com.oracle.database.jdbc:ojdbc8,com.ibm.db2:jcc,com.microsoft.sqlserver:mssql-jdbc
+  package:
+    type: mutable-jar
+    user-providers-directory: providers
+  http:
+    auth:
+      basic: true
+      policy:
+        main-policy:
+          roles-allowed: admin
+      permission:
+        main:
+          paths: /*
+          policy: main-policy
+        public:
+          paths: /q/health/*
+          policy: permit
+          methods: GET
   security:
-    security-domains:
-      pim:
-        classic-authentication:
-          login-modules:
-            UsersRoles:
-              code: UsersRoles
-              flag: required
-              module-options:
-                usersProperties: application-users.properties
-                rolesProperties: application-roles.properties
-kieservers:
-  - host: http://localhost:8080/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD
-  - host: http://localhost:8280/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD1
+    users:
+      file:
+        realm-name: pim_file
+        enabled: true
+        plain-text: true
+        users: users.properties
+        roles: roles.properties
+    jdbc:
+      realm-name: pim_jdbc
+      enabled: true
+      principal-query:
+        sql: SELECT u.password, u.role FROM users u WHERE u.username=?
+    ldap:
+      realm-name: pim_ldap
+      enabled: true
+      dir-context:
+        url: ldap://override-when-needed
+      identity-mapping:
+        search-base-dn: ou=users,o=YourCompany,c=ES
+# Quartz configuration
+  quartz:
+    store-type: ram
+    start-mode: forced
+  resteasy:
+    path: /rest
+  datasource:
+    db-kind: h2
+    jdbc:
+      url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password: sa
+  hibernate-orm:
+    database:
+      generation: update
+pim:
+  auth-method: file
+"%int":
```

Comment:
remove this from the docs as this profile is only used for integration tests

Reply:
Removed